### PR TITLE
etcdctl: Add aliases for command flags.

### LINKED ISSUE #4866: etcdctl short flags for --recursive please.
Can we get -r to alias --recursive with etcdctl?

```bash
etcdctl ls -r /skydns/local
```

Thanks

agokjr

### PR DESCRIPTION
**Pull request by ajityagaty**
Add aliases to the flags that are supplied to the sub commands.

Fixes #4866

### CODE REVIEW

**Comment by ajityagaty** [comment]
@xiang90 I have added aliases to some of the flags. I have not added aliases for flags that have 2 words. Please let me know if anything needs to be changed/added or if there is any specific format that needs to be followed.

**Comment by ajityagaty** [comment]
SemaphoreCI failure seems unrelated.

File: etcdctl/command/ls_command.go [hunk 1]
```diff
@@ -27,10 +27,10 @@ func NewLsCommand() cli.Command {
 		Usage:     "retrieve a directory",
 		ArgsUsage: "[key]",
 		Flags: []cli.Flag{
-			cli.BoolFlag{Name: "sort", Usage: "returns result in sorted order"},
-			cli.BoolFlag{Name: "recursive", Usage: "returns all key names recursively for the given path"},
+			cli.BoolFlag{Name: "sort, s", Usage: "returns result in sorted order"},
```
Comment (xiang90) [inline etcdctl/command/ls_command.go]: i think sort is ok.

File: etcdctl/command/import_snap_command.go [hunk 2]
```diff
@@ -40,8 +40,8 @@ func NewImportSnapCommand() cli.Command {
 		Usage:     "import a snapshot to a cluster",
 		ArgsUsage: " ",
 		Flags: []cli.Flag{
-			cli.StringFlag{Name: "snap", Value: "", Usage: "Path to the valid etcd 0.4.x snapshot."},
-			cli.StringSliceFlag{Name: "hidden", Value: new(cli.StringSlice), Usage: "Hidden key spaces to import from snapshot"},
+			cli.StringFlag{Name: "snap, s", Value: "", Usage: "Path to the valid etcd 0.4.x snapshot."},
```
Comment (xiang90) [inline etcdctl/command/import_snap_command.go]: snap, hidden are probably ok. not worth to give them a short flag i guess.

File: etcdctl/command/ls_command.go (hunk 1 above)
Comment (ajityagaty) [inline etcdctl/command/ls_command.go]: @xiang90 You mean we need not add an alias for 'sort'?

Comment (xiang90) [inline etcdctl/command/ls_command.go]: yea. i think sort is short enough.

**Comment by ajityagaty** [comment]
@xiang90 We should probably add an alias if the flag is more than 4 letters? If we stick to that then there are some more flags for which I need to remove the aliases.

**Comment by xiang90** [comment]
@ajityagaty I feel so.

**Comment by ajityagaty** [comment]
@xiang90 I have removed aliases for all flags that are <= 5 in length.

Also, the SemaphoreCI error seems unrelated.

**Comment by xiang90** [comment]
LGTM.

@ajityagaty ci failure is tracked at https://github.com/coreos/etcd/issues/4889

**Comment by ajityagaty** [comment]
@xiang90 Thanks !

**Pull request merged**